catalog/core/catalog-core-camelcomponent/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -15,32 +15,34 @@
            xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0"
            xsi:schemaLocation="
   http://www.osgi.org/xmlns/blueprint/v1.0.0 http://www.osgi.org/xmlns/blueprint/v1.0.0/blueprint.xsd"
-        >
+>
 
     <reference id="transformerMapper" interface="ddf.mime.MimeTypeToTransformerMapper"/>
 
     <reference id="mimeTypeMapper" interface="ddf.mime.MimeTypeMapper"/>
 
-    <reference id="framework" interface="ddf.catalog.CatalogFramework"/>
+    <reference id="framework" interface="ddf.catalog.CatalogFramework" availability="optional"/>
```

Comment:
I agree, I was pretty hesitant about making this change. What's the best way to handle this gracefully? Null checks?